module6/lessons/Week5/PreppingForInterviews.md

```diff
@@ -0,0 +1,102 @@
+---
+layout: page
+title: Prepping for an Interview
+---
+
+You've busted your butt and landed an interview! GREAT! Now what?
+
+Each interview deserves its own preparation.  Try to give yourself at least a few days before the interview so that you are fully prepared and confident on the day.
+
+# Prepping for an Interview - A Checklist!
```

Comment:
I think this list is great!

I'm not sure where to add this, but I think worth adding a note about it being totally ok to ask the recruiter what format of interview to expect. I've heard from a couple of students that last week they were expecting a different format (either a code challenge or all STAR). Apparently Isiah has been doing a lot of edibit practice over break after the disaster interview with you.

Reply:
Glad that Isiah has been practicing!  

I'm a bit surprised that some students were caught off-guard 🙃 In our PD session on Monday, James asked what the Thursday interview would be like, and we even went over how you could incorporate STAR into a technical question 😆

Reply:
I'm going to add an initial step to confirm the interview-type!